docker/packager/packager

```diff
@@ -77,6 +77,7 @@ def pre_build(repo_path: Path, env_variables: List[str]) -> None:
 def run_docker_image_with_env(
     image_name: str,
     as_root: bool,
+    additional_scripts: Path,
```

Comment:
do we really need it? What about just checking some particular `/ClickHouse/packages/{pre,post}-build` directories in the `docker/packager/binary/build.sh` script? I feel it overcomplicating.

Reply:
It depends on how we would like to execute the `packager` scripts e.g. from the directory where it lies or from another one. In the latter case better to specify absolute path for all the volumes and that's why I made all the paths configurable.